Sming/Makefile

```diff
@@ -280,6 +288,11 @@ $(USER_LIBDIR)/libaxtls.a:
 	$(Q) $(MAKE) -C third-party/axtls-8266 -e V=$(V) BIN_DIR="$(SMING_HOME)/$(USER_LIBDIR)"
 endif
 
+$(USER_LIBDIR)/libmainmm.a: $(addprefix $(SDK_LIBDIR)/,libmain.a)
```

Comment:
That is not necessary, but will make things easier to follow/understand. So I will add the change in a minute...